.circleci/setup_storage.sh

```diff
@@ -35,11 +35,11 @@ echo $(GREEN "Setting up Cloud SDK environment...")
 (set -x && echo "source ~/google-cloud-sdk/path.bash.inc" >> $BASH_ENV)
 source $BASH_ENV
 
-echo $(GREEN "Extracting credentials...")
-openssl aes-256-cbc -k $GCP_TOKEN -in ./build-system/common/sa-travis-key.json.enc -out sa-travis-key.json -d
+echo $(GREEN "Writing key to json file...")
+echo $STORAGE_TOKEN > storage-key.json
 
 echo $(GREEN "Authenticating with GCP storage...")
-gcloud auth activate-service-account --key-file=sa-travis-key.json
+gcloud auth activate-service-account --key-file=storage-key.json
```

Comment:
Yay, logs are less noisy than before: https://app.circleci.com/pipelines/github/ampproject/amphtml/2973/workflows/ef569347-0472-48d6-aba3-86fddf2a168e/jobs/34847/parallel-runs/0/steps/0-104

Any idea how to get rid of the access warning and the survey messages?

Reply:
Got rid of a few, not the survey though.